renderdoccmd/renderdoccmd_android.cpp

```diff
@@ -38,7 +38,7 @@ extern "C" {
 #include <android/log.h>
 #define LOGCAT_TAG "renderdoc"
 
-using std::string;
+using namespace std;
```

Comment:
I'd prefer not bringing in the entire namespace here, even if it is just an isolated cpp file. You should just be able to get away with vector, string and istringstream from what I can see.